datacenter/ucp/2.1/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -13,7 +13,18 @@ The next step is creating a backup policy and disaster recovery plan.
 
 ## Backup policy
 
-As part of your backup policy you should regularly create backups of UCP.
+As part of your backup policy you should regularly create backups of swarm and
+UCP.
+
+When you create a UCP backup, you're only saving:
+
+* Users, teams, and permissions.
```

Comment:
Not sure about the granularity of this list. Maybe some rephrasing would make the list items more "even" with each other. Just a nit.